pycytominer/cyto_utils/cp_aggregate.py

```diff
@@ -0,0 +1,138 @@
+        .count()
+        .reset_index()
+        .rename(columns={f"{fields_of_view_feature}": f"Metadata_Site_Count"})
+    )
+
+    return fields_count_df
+
+
+def aggregate_image_count_features(df, image_features_df, image_cols, strata):
+    """Aggregate the Count features in the Image table.
+
+    Parameters
+    ----------
+    df : pandas.core.frame.DataFrame
+        Dataframe of aggregated profiles.
+    image_features_df : pandas.core.frame.DataFrame
+        Image table dataframe with Count features
+    image_cols : list of str
+        Columns to select from the image table.
+    strata :  list of str
+        The columns to groupby and aggregate single cells.
+
+    Returns
+    -------
+    df : pandas.core.frame.DataFrame
+        DataFrame with aggregated Count features in the Image table.
+    remove_cols : list of str
+        Columns to remove from the image table before aggregating using aggregate_image_features()
+    """
+
+    count_features = list(
+        image_features_df.columns[
+            image_features_df.columns.str.startswith("Image_Count")
+        ]
+    )
+
+    remove_cols = list(np.union1d(image_cols, count_features))
+    keep_cols = list(np.union1d(strata, count_features))
+    count_df = image_features_df[keep_cols].copy()
+    count_df = count_df.groupby(strata, dropna=False).sum().reset_index()
+    df = df.merge(count_df, on=strata, how="left")
+
+    return df, remove_cols
+
+
+def aggregate_image_features(
+    df,
+    image_features_df,
+    image_feature_categories,
+    image_cols,
+    strata,
+    aggregation_operation,
+):
+    """Aggregate the non-Count image features.
+
+    Parameters
+    ----------
+    df : pandas.core.frame.DataFrame
+        Dataframe of aggregated profiles.
+    image_features_df : pandas.core.frame.DataFrame
+        Image table dataframe with all the image_feature_category features.
+    image_feature_categories : list of str
+        List of categories of features from the image table to add to the profiles.
+    image_cols : list of str
+        Columns to select from the image table.
+    strata :  list of str
+        The columns to groupby and aggregate single cells.
+    aggregation_operation : str
+        Operation to perform image table feature aggregation.
+
+    Returns
+    -------
+    df : pandas.core.frame.DataFrame
+        DataFrame of aggregated image features.
+
+    """
+
+    if "Count" in image_feature_categories:
+        df, remove_cols = aggregate_image_count_features(
+            df, image_features_df, image_cols, strata
+        )
+    else:
+        remove_cols = list(image_cols) + list(
+            image_features_df.columns[
+                image_features_df.columns.str.startswith("Image_Count")
+            ]
+        )
+
+    if (
+        not len(np.setdiff1d(image_feature_categories, ["Count"])) == 0
+    ):  # The following block will not be run if the input image category is only "Count"
```

Comment:
This seems necessary, and CellProfiler specific. Can you add more documentation to specify the circumstances that this might occur?

Also, please move the in-line comment on line 122 to above the if statement. PEP8 says [inline comments should be used sparingly](https://www.python.org/dev/peps/pep-0008/#inline-comments), and typically I think of inline comments as being very short.